### ISSUE TITLE
Error when build

### ISSUE THREAD

**Issue by kibot914** [GameAnalytics/GA-SDK-DEFOLD#15]
error: range-based 'for' loops are not allowed in C++98 mode

How can I solve this error when I bundle android apk?

**Comment by subsoap** [comment]
@kibot914 can you test bundling your project with version 1.2.152 of bob.jar?

http://d.defold.com/archive/11b1e7662dd68172fca551c52cba248eea16a364/bob/bob.jar

**Comment by makkil** [comment]
the problem happens when compiling in 64-bit android

it is probably missing in ext.manifest

```
armv64-android:
         context:
             flags: ["-std = c ++ 11"]
```

**Comment by the1schwartz** [comment]
Thanks this has been fixed now in the latest version (v2.2.3)
